extensions/cheetah/src/lib/constant.ts

```diff
@@ -0,0 +1,31 @@
+import { join } from "path";
+import { HOME_PATH } from "cheetah-core";
+
+export const refreshKeyword = "[refresh]";
+
+export const cachePath = join(
```

Comment:
You should use [environment.supportPath](https://developers.raycast.com/api-reference/environment#example) which is made for your extension's files instead.